### DISCUSSION TITLE
Ability to open a new tab within a split view

### DISCUSSION THREAD

**Discussion by LeMoonStar** [zen-browser/desktop#2137] [Ideas]
Currently, adding a new tab to a split view is annoying - you either have to split a link and use that tab, or re-create the split view with more tabs selected. Instead, there should be the ability to open a new tab in the current split view, **both as a button, and as a keyboard shortcut.**

As the split view shortcuts already use `CTRL+ALT`, and `CTRL+ALT+T` is not used so far, it seems like a perfect choice for the default shortcut.

**Comment by spectrum705** [comment]
also having a way to add any opened tab to existing split view would be very useful. currently we need to unsplit the whole thing

**Comment by zahin-mohammad** [comment]
The way arc implements this feature is very intuitive! I would love for `zen` to do the same.

**Comment by tedbergstrand** [comment]
Agreed. Split view is fantastic, but getting things in and out of it feels clunky. 

My two cents:

- There should be a modifier key to open a new tab in a split. (Maybe there is, but I've only been able to find it in the right click menu). Something like Alt+Middle Click
  - Maybe when the modifier key (or combo) is held, a small close button could appear at the top of each tab in the grid, allowing it to be removed specifically from the Split without closing the tab.
- You should be able to drag tabs from the sidebar in and out of the Split. Right now, it's a bit annoying to manage splits with a "master tab" (that doesn't change) and other "working tabs" (that do change). 
  - Maybe this would entail some kind of "folderizing" for the Splits, so tabs could easily be dragged in and out.
  
Working in the tab bar is somewhat inconvenient on ultra-wide monitors because you have to mouse to the far left, make your adjustments, and then go back to whatever you were working on. So while I'd love to see some more intuitive options in the tab bar, I'd also really like some "in-tab" control, too.

I really love Zen Browser, though. Thanks so much for your hard work.

**Comment by aTosser** [comment]
Just an option to change middle-click behavior to open in split.

or an option to make all new tabs open in split view. (think tiling WM)

**Comment by nrbnlulu** [comment]
`CTRL+ALT+T` is open terminal on linux

**Comment by XanderLeaDaren** [comment]
Just drag-and-drop any other tab onto the already split view grid, and voilà!

**Reply by nrbnlulu** [reply]
Free the mice 😁

**Comment by willbonney** [comment]
This would be a really nice addition.

**Comment by JordanTranchina** [comment]
Vibecoded this feature here, if anyone is able to test it. https://github.com/zen-browser/desktop/pull/9721
